tests/infrared/metrics-collectd-qdr.yaml.template

```diff
@@ -4,28 +4,47 @@ tripleo_heat_templates:
 
 custom_templates:
     parameter_defaults:
-        CollectdAmqpInterval: 1
-        CollectdDefaultPollingInterval: 1
-        CollectdExtraPlugins:
+        CollectdAmqpInterval: 5
+        CollectdDefaultPollingInterval: 5
+        CollectdDefaultPlugins:
             - cpu
             - df
+            - disk
             - hugepages
+            - intel_rdt
+            - interface
+            - ipmi
+            - load
+            - memory
             - ovs_events
             - ovs_stats
-            - load
+            - processes
+            - unixsock
             - uptime
         DnsServers: ["10.19.42.41","10.11.5.19","10.5.30.160"]
         ExtraConfig:
-            collectd::plugin::cpu::valuespercentage: true
+            collectd::plugin::cpu::interval: 5
             collectd::plugin::cpu::reportbycpu: true
             collectd::plugin::cpu::reportbystate: true
             collectd::plugin::cpu::reportnumcpu: false
+            collectd::plugin::cpu::valuespercentage: true
+            collectd::plugin::df::interval: 300
             collectd::plugin::df::ignoreselected: true
+            collectd::plugin::df::fstypes: ['xfs']
```

Comment:
The only xfs mount points the container sees are bind-mounted config files (lots of them!) that each present 6 metrics that are identical to the rootfs ones represented separately (type: overlay). This plugin doesn't do a good job of reporting the df situation on host file systems other than root.